WeakAurasOptions/LoadOptions.lua

```diff
@@ -619,13 +619,15 @@ function OptionsPrivate.ConstructOptions(prototype, data, startorder, triggernum
             values = WeakAuras[arg.values];
           end
         end
+        local desc = arg.desc and OptionsPrivate.Private[arg.desc] or WeakAuras[arg.desc] or arg.desc or nil
```

Comment:
Please make this simply desc = arg.desc.

The lookup into Private and WeakAuras is something that is not used for any other description and argueably is a anti-pattern anyway.

Reply:
I wanted to be able to define it once somewhere, rather than paste it in to a few places making it awkward to update later.  Types seemed like a good place. `Private.stringName` and `WeakAuras.stringName` didn't appear to be accessible inside Prototypes where the `args` table is made, and I thought this would allow for both simple strings or Types references to be used on Select or Unit controls in future. 

Happy to not go that route with this stuff. Which point do you want the change made? Just paste the string several times for simplicity?